src/component/base/DraftEditorProps.js

```diff
@@ -128,4 +129,7 @@ export type DraftEditorProps = {
   // Provide a map of inline style names corresponding to CSS style objects
   // that will be rendered for matching ranges.
   customStyleMap?: Object,
+
+  // custom block maps
```

Comment:
Could be helpful to have this documentation be a little more fleshed out. I think we'll definitely want a proper article on draftjs.org too, to really clarify what all this configuration is, but a little blurb here would be good too.